docs/windmill_dark/footer.html

```diff
@@ -81,7 +81,7 @@ <h4>Community</h4>
       <div class="col-sm-6 text-right footnote-copy">
         <ul class="list-unstyled list-inline">
           <li>© 2020 Microsoft</li>
```

Comment:
Hey Chris, orthogonal to your PR, but since you are editing this file could you update this bit? Or perhaps we can add some JS to update it dynamically? :)

Reply:
Done.

Reply:
Nice!